### ISSUE TITLE
Include container name in scale output when creating a container

### ISSUE THREAD

**Issue by jianbin-netskope** [docker/compose#2891]
I first run `up` command and later use `scale` to increase the number of services.  The output is inconsistent.  The new containers are not shown in the output either.
```
docker-compose -f kafka-cluster.yml scale zookeeper=2 kafka=3
Starting docker_zookeeper_1 ... done
Creating and starting 2 ... done
Starting docker_kafka_1 ... done
Creating and starting 2 ... done
Creating and starting 3 ... done
```

The output would be consistent and more descriptive as 
```
docker-compose -f kafka-cluster.yml scale zookeeper=2 kafka=3
Starting docker_zookeeper_1 ... done
Creating and starting docker_zookeeper_2 ... done
Starting docker_kafka_1 ... done
Creating and starting docker_kafka_2 ... done
Creating and starting docker_kafka_3 ... done
```

**Comment by jianbin-netskope** [comment]
Later after `stop` and then `scale` gives more consistent and descriptive output
```
docker-compose -f kafka-cluster.yml scale zookeeper=2 kafka=3
Starting docker_zookeeper_1 ... done
Starting docker_zookeeper_2 ... done
Starting docker_kafka_1 ... done
Starting docker_kafka_2 ... done
Starting docker_kafka_3 ... done
```

**Comment by dnephin** [comment]
By "new containers are not shown in the output" I assume you mean log output. That is covered by #1115

I'm going to update the title of this issue to better reflect the requested change.

**Comment by JesusTinoco** [comment]
I think this could fix that issue #3036

**Comment by JesusTinoco** [comment]
#dibs

**Comment by dnephin** [comment]
This was fixed by #3036
